app/views/assets/publishing/requested_approval_assets.html.erb

```diff
@@ -66,4 +69,47 @@
         </p>
     <% end %>
 <% end %>
-</div>
\ No newline at end of file
+  <% unless @rejected_assets.empty? %>
+    <a class="rejected_items" href="#" style="display:block;" onclick="toggleRejected()">Show rejected</a>
+    <a class="rejected_items" href="#" style="display:none;" onclick="toggleRejected()">Hide rejected</a>
+    <li class="rejected_items" style="display:none;">
+      <h3 style="margin:8px 0px;">Items with rejected status:</h3>
+      <% @rejected_assets.each do |asset| %>
+        <ul class="item_waiting_for_approval">
+          <li class="type_and_title">
+            <b><%= text_for_resource asset -%> : </b>
+            <%= link_to(asset.title, asset,option={target: :_blank}) -%>
+            <%= list_item_visibility(asset) -%>
+          </li>
+          <div class="request_info">
+            Requested by
+            <% asset.publish_requesters.each do |requester| %>
+              <%= link_to(requester.name, requester, option={target: :_blank}) %>
+              on
+              <%= date_as_string(asset.resource_publish_logs.last(2).first.created_at,true) %>
+            <% end %>
+            <br/>
+            Rejected on:&nbsp;
+            <%= date_as_string(asset.last_publishing_log.created_at,true) %>
+
+            Comments: <%= asset.last_publishing_log.comment %>
+          </div>
+        </ul>
+      <% end %>
+    </li>
+  <% end %>
+</div>
+
+<script>
+    function toggleRejected() {
+        event.preventDefault();
+        let rejected_items = $j('.rejected_items');
+        for(let rejected_element of rejected_items) {
+            if ($j(rejected_element).is(':hidden')) {
+                rejected_element.style.display = "block";
+            } else {
+                $j(rejected_element).hide();
+            }
+        }
```

Comment:
infact, you can just use jquery `toggle `- https://api.jquery.com/toggle/

this whole block of javascript could be probably be replaced simply with `$j('.rejected_items').toggle();`

Reply:
That is much better, thanks!

Resolved in [fca47a6](https://github.com/seek4science/seek/pull/1484/commits/fca47a6efd6539ec30116c006037ec5ad000fb43)